invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/macros/detail.html

```diff
@@ -213,3 +213,19 @@ <h2>{{add_description.type.title_l10n }} <span class="text-muted language">{{ '(
     {% endif %}
   {% endfor %}
 {% endmacro %}
+
+{% macro show_detail_conference(conference) %}
+     <dd>
+       {%- if conference.url %}
+        <a href="{{conference.url}}"><i class="fa fa-external-link"></i> {{conference.title}}</a>
+       {%- else %}
+        {{conference.title}}
+       {%- endif%}
+       {%- if conference.place %}, {{conference.place}} {%- endif %}
+       {%- if conference.dates %}, {{conference.dates}} {%- endif %}
+       {% if conference.session %}(Session {{conference.session}}{% if conference.session_part %}, Part {{conference.session_part}}{%- endif %}){%- endif %}
+     </dd>
+     {%- if conference.url and not conference.title %}
```

Comment:
Question: Why can't we display both, conference title and url?

Reply:
This was taken as it is from [Zenodo](https://github.com/zenodo/zenodo/blob/master/zenodo/modules/records/templates/zenodo_records/box/info.html#L183)